src/subdomains/supporting/fiat-output/fiat-output-job.service.ts

```diff
@@ -0,0 +1,261 @@
+import { forwardRef, Inject, Injectable } from '@nestjs/common';
+import { CronExpression } from '@nestjs/schedule';
+import { Config } from 'src/config/config';
+import { AzureStorageService } from 'src/integration/infrastructure/azure-storage.service';
+import { AssetType } from 'src/shared/models/asset/asset.entity';
+import { AssetService } from 'src/shared/models/asset/asset.service';
+import { CountryService } from 'src/shared/models/country/country.service';
+import { DfxLogger } from 'src/shared/services/dfx-logger';
+import { Process } from 'src/shared/services/process.service';
+import { DfxCron } from 'src/shared/utils/cron';
+import { Util } from 'src/shared/utils/util';
+import { LiquidityManagementBalanceService } from 'src/subdomains/core/liquidity-management/services/liquidity-management-balance.service';
+import { In, IsNull, Not } from 'typeorm';
+import { BankTx, BankTxType } from '../bank-tx/bank-tx/entities/bank-tx.entity';
+import { BankTxService } from '../bank-tx/bank-tx/services/bank-tx.service';
+import { BankService } from '../bank/bank/bank.service';
+import { LogService } from '../log/log.service';
+import { Ep2ReportService } from './ep2-report.service';
+import { FiatOutput, FiatOutputType } from './fiat-output.entity';
+import { FiatOutputRepository } from './fiat-output.repository';
+
+@Injectable()
+export class FiatOutputJobService {
+  private readonly logger = new DfxLogger(FiatOutputJobService);
+
+  constructor(
+    private readonly fiatOutputRepo: FiatOutputRepository,
+    @Inject(forwardRef(() => BankTxService))
+    private readonly bankTxService: BankTxService,
+    private readonly ep2ReportService: Ep2ReportService,
+    private readonly bankService: BankService,
+    private readonly countryService: CountryService,
+    private readonly liquidityManagementBalanceService: LiquidityManagementBalanceService,
+    private readonly assetService: AssetService,
+    private readonly logService: LogService,
+  ) {}
+
+  @DfxCron(CronExpression.EVERY_MINUTE, { process: Process.FIAT_OUTPUT, timeout: 1800 })
+  async fillFiatOutput() {
+    await this.fillPreValutaDate();
+    await this.fillIsReadyDate();
+    await this.setBatchId();
+    await this.setDates();
+    await this.bankTxSearch();
```

Comment:
The job names are not really accurate 🤔  maybe they should be more about the included logic, e.g. more like:
- assignBankAccount
- setReady
- createBatches
- checkTransmission
- searchOutgoingBankTx